core/src/test/resources/server.conf

```diff
@@ -154,7 +154,7 @@ line.tcp.n.updates.per.load.balance=100000
 line.tcp.max.load.ratio=1.5
 line.tcp.maintenance.job.interval=1000
 line.tcp.default.partition.by=MONTH
-line.tcp.io.aggressive.recv=true
+line.tcp.aggressive.read.retry.count=10000
```

Comment:
In reading in PropServerConfiguration.java default is 0, here default is 1000, I believe they should be same for clarity

Reply:
This is test conf file, it doesn't contain defaults.  It is there to make sure we pick up values from file correctly. So all values in this file are different to default :)

Reply:
I'd expect them to be the same to document the defaults to the user and defaults are already optimal.

Reply:
this is a test file `core/src/TEST/resources/server.conf` it saves no other purpose than to check that settings can be successfully read from the file. The values are deliberately different from the defaults for every setting in this file, not just the line in question.

If this file has same values as default, test won't be able to distinguish where value is coming from, default or file.

I take you point that perhaps `server.conf` that we do deploy should have correct deploy, but that's another file and another change